staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -669,6 +669,14 @@ func (r *crdHandler) getOrCreateServingInfoFor(uid types.UID, name string) (*crd
 		openAPIModels = nil
 	}
 
+	var typeConverter internal.TypeConverter = internal.DeducedTypeConverter{}
```

Comment:
That can't compile, you can't import `internal` from a different package structure.

Reply:
It means you have to move the `TypeConverter` outside of internal first.